apps/crn-server/test/data-providers/reminders.data-provider.test.ts

```diff
@@ -564,6 +565,94 @@ describe('Reminder Data Provider', () => {
       });
     });
 
+    describe('Video upserted in Event Reminder', () => {
+      beforeAll(() => {
+        jest.useFakeTimers('modern');
+      });
+
+      afterAll(() => {
+        jest.useRealTimers();
+      });
+
+      beforeEach(async () => {});
+
+      test('Should fetch the reminder if its video was upserted in the last 24 hours', async () => {
```

Comment:
actually if the video was `inserted` as part of the Event payload (as in - if the event was created with the video materials already in there) - this field wouldn't be populated (see: squidex create script - it's empty) so I think saying `upserted` here is a little misleading